lib/src/flutter_search_bar_base.dart

```diff
@@ -46,8 +46,8 @@ class SearchBar {
   /// A callback which is invoked each time the text field's value changes
   final TextFieldChangeCallback onChanged;
 
-  /// Whether or not the search bar should show up number keyboard. Defaults to 'false'.
-  final bool numberKeyboard;
+  /// What the keyboard type on the search bar. Defaults to 'TextInputType.text'.
```

Comment:
This sentence doesn't really make grammatical sense, can you change it to this text I edited from the flutter docs?

`The type of keyboard to use for editing the search bar text. Defaults to 'TextInputType.text'.`

Reply:
I fixed in 6397f08